From 549912a95578aace88ddf38b1188e842f781ad38 Mon Sep 17 00:00:00 2001
From: Rajesh Gugulothu <gugulothu.rajesh@xilinx.com>
Date: Tue, 27 Oct 2020 10:07:07 -0700
Subject: [PATCH 1733/1852] v4l: xilinx: update the bits per component

commit e04d2546fd5e6015fe6e81dc6724c6b67d100267 from
https://github.com/Xilinx/linux-xlnx.git

update the bits per component varilable based on the received stream.

Signed-off-by: Rajesh Gugulothu <gugulothu.rajesh@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-dprxss.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/media/platform/xilinx/xilinx-dprxss.c b/drivers/media/platform/xilinx/xilinx-dprxss.c
index dde0c2fcda26..9b78f83fce3e 100644
--- a/drivers/media/platform/xilinx/xilinx-dprxss.c
+++ b/drivers/media/platform/xilinx/xilinx-dprxss.c
@@ -481,6 +481,7 @@ static int xdprxss_get_stream_properties(struct xdprxss_state *state)
 	read_val = xdprxss_read(state, XDPRX_DTG_REG);
 	xdprxss_write(state, XDPRX_DTG_REG, (read_val | 0x1));
 	fmt = FIELD_GET(XDPRX_MSA_FMT_MASK, rxmsa_misc);
+	state->bpc = FIELD_GET(XDPRX_MSA_BPC_MASK, rxmsa_misc);
 
 	switch (fmt) {
 	case XDPRX_COLOR_FORMAT_422:
-- 
2.31.1

